docs/application/dotnet/guides/nui/text.md

```diff
@@ -24,15 +24,15 @@ The `TextLabel` class is lightweight, non-editable, and do not respond to user i
 
 To create a text label:
 
-1.  Create an instance of the [Tizen.NUI.BaseComponents.TextLabel](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextLabel.html) class and define the label text as a parameter:
+1.  Create an instance of the `TextLabel` class and define the label text as a parameter. To make text visible set `TextColor` property to `Color.White`:

```

Comment:
Suggestion: Please add a comma after visible

1.  Create an instance of the `TextLabel` class and define the label text as a parameter. To make text visible, set `TextColor` property to `Color.White`: